lib/components/primitive-components/Group/Group.ts

```diff
@@ -874,6 +874,46 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
     }
   }
 
+  _getPresetConfig(
```

Comment:
(this should not be an instance method, can be moved to separate file `getPresetAutoroutingConfig`

```suggestion
  _getPresetAutoroutingConfig(
```